### PR TITLE
visualise small bboxes

### PR DESCRIPTION
**Pull request by RibenaMapleSyrup**
I've adjusted draw_data.py to improve visualisation of bboxes and adjusted imageio.py to improve resolution of image subplots.

### CODE REVIEW

File: icevision/utils/imageio.py [hunk 1]
```diff
@@ -27,11 +27,10 @@ def show_img(img, ax=None, show: bool = False, **kwargs):
 
 
 def plot_grid(fs: List[callable], ncols=1, figsize=None, show=False, **kwargs):
-    figsize = figsize or (12 * len(fs) / ncols, 12)
     nrows = math.ceil(len(fs) / ncols)
+    figsize = (12 * ncols, 12 * nrows)
```
Comment (lgvaz) [inline icevision/utils/imageio.py:31]:
```suggestion
    figsize = figsize or (12 * ncols, 12 * nrows)
```

**Review by lgvaz — CHANGES_REQUESTED** [review]
Hello there Sean! First of all thanks a lot for the PR :heart: 

I made a commit to your branch fixing a merge conflict, can you check I made the correct changes? :sweat_smile: 

The code looks perfect! I just added one suggestion, tell me what you think =)

**Comment by lgvaz** [comment]
Sorry for the delay btw, for some reason in my mind I was sure I merged this already hahaha

**Comment by lgvaz** [comment]
The test failing with python 3.6 is not related to this PR btw, don't worry

File: icevision/utils/imageio.py (hunk 1 above)
Comment (RibenaMapleSyrup) [reply icevision/utils/imageio.py:31]: Looks great lgvaz! Thanks

**Review by lgvaz — APPROVED** [review]
Perfecto!! Let's go!

**Pull request merged**